.travis.yml

```diff
@@ -66,9 +72,16 @@ script:
   - make analyze --keep-going
   - if [[ $PHPDBG != 1 ]]; then xdebug-enable; fi
   - if [[ $PHPDBG != 1 ]]; then $PHPUNIT_BIN; else phpdbg -qrr $PHPUNIT_BIN; fi
-  - ./tests/e2e_tests
+  - |
+    if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then
+      ./tests/e2e_tests bin/infection;
+    else
+      make build/bin/infection.phar;
```

Comment:
about `composer update` - I'm afraid this can be dangerous

1. we want PHAR to be very stable, that's why we have `composer.lock` committed
2. by stability I mean not only passed tests and the green build, but predictable console output and everything else

we saw situations where upgraded dependencies led to different bugs like e.g. new Differ outputs worse diff, etc.

So I prefer to have `composer.lock` and ship exactly what we know works as expected